sdk/storage/perf-tests/storage-blob/test/storageTest.spec.ts

```diff
@@ -27,10 +27,13 @@ export abstract class StorageBlobTest<TOptions> extends PerfTest<TOptions> {
       getValueInConnString(connectionString, "AccountName"),
       getValueInConnString(connectionString, "AccountKey")
     );
-    this.blobServiceClient = BlobServiceClient.fromConnectionString(
-      connectionString,
-      this.configureClientOptionsCoreV1({})
-    );
+    this.blobServiceClient = BlobServiceClient.fromConnectionString(connectionString);
+    const options = this.configureClientOptions({ additionalPolicies: [] });
+
+    const pipeline = this.blobServiceClient["storageClientContext"].pipeline;
```

Comment:
Hmmm, I can see why you're having to do this.

Why doesn't storage's `client options` extend from common client options? 
That way we'd get the `additionalPolicies` array directly.

Reply:
Storage never used our common options because they had some custom ones, and now it would conflict a bit since they have the old "HttpClient" shape from core-http that is maintained for back compat. :(